src/portal/components/CreateIP.vue

```diff
@@ -93,7 +93,7 @@ export default class CreateIP extends Vue {
     this.$emit("create", this.publicIP, this.gateway);
   }
   ipcheck() {
-    const ipRegex = new RegExp("^(?:[0-9]{1,3}.){3}[0-9]{1,3}$");
+    const ipRegex = new RegExp("^(?:[0-9]{1,3}.){3}[0-9]{1,3}/(16|24)$");
```

Comment:
the mask length can be 32 as it may be just one ip
```suggestion
    const ipRegex = new RegExp("^(?:[0-9]{1,3}.){3}[0-9]{1,3}/(16|32)$");
```